test/it/crdvalidation/etcdopstask/helper.go

```diff
@@ -0,0 +1,83 @@
+// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package etcdopstask
```

Comment:
why are all these test in a different package?
These are specific to `EtcdOpsTask` IT tests and should be together with those.